server/src/test/java/org/candlepin/policy/PoolRulesTest.java

```diff
@@ -767,11 +774,12 @@ public void standaloneVirtLimitSubCreate() {
         assert ("true".equals(unmappedVirtPool.getAttributeValue("virt_only")));
         assert ("true".equals(unmappedVirtPool.getAttributeValue("unmapped_guests_only")));
 
-        assertProvidedProducts(s.getProvidedProducts(),
+        // The derived provided products of the sub should be promoted to provided products
+        // on the unmappedVirtPool
+        assertProvidedProducts(s.getDerivedProvidedProducts(),
```

Comment:
The change looks to hit both create and update pools, but the test appears to only be covering create. (and the test passes even if I comment out important parts of the patch.